APIv1.yaml

```diff
@@ -0,0 +1,254 @@
+openapi: 3.0.3
+info:
```

Comment:
@datenangebot Did you check with @provokateurin about generating those from the controller annotations? I'd be in favour of going the automated approach directly if possible.